webdriver/tests/new_session/headers.py

```diff
@@ -0,0 +1,6 @@
+def test_missing_first_match(new_session, add_browser_capabilities):
+    response, _ = new_session({"capabilities": {"alwaysMatch": add_browser_capabilities({})}})
+    assert 'cache-control' in response.headers
```

Comment:
I think you should lowercase the recieved values here.

Reply:
@jgraham see PR question: shall we by default lowercase key and value pairs since headers are case insensitive?